service-workers/service-worker/fetch-event.https.html

```diff
@@ -6,6 +6,9 @@
 <body>
 <script>
 var worker = 'resources/fetch-event-test-worker.js';
+function wait(ms) {
+    return new Promise(resolve => step_timeout(resolve, ms));
```

Comment:
is it ok to call step_timeout without a test object, like test.step_timeout?

Reply:
I think so, it's exposed explicitly.